MangoAPI.Client/src/environments/environment.ts

```diff
@@ -1,4 +1,6 @@
 export const environment = {
   production: false,
-  baseUrl: 'https://localhost:5001/'
-};
+  baseUrl: 'https://localhost:5001/',
+  blobStorageUrl: 'http://127.0.0.1:10000/',
+  sasToken: '?sv=2018-03-28&st=2023-02-04T22%3A00%3A19Z&se=2023-02-21T22%3A00%3A00Z&sr=c&sp=racwdl&sig=X19cS5b3%2B0MAFzJviGax7U1pZWPJiGdPT%2BWIyvtWmno%3D'
```

Comment:
The problem here is that we must keep SAS in source control for prod as well, so it breaks security. Instead, I propose to use `FormFile` approach, so

1) If fileform is any: send file to the `api/documents` POST endpoint,
2) Response from `api/documents` contains URL of newly uploaded blob file
3) Send message with new file URL attached
4) Display image on frontend (as you have already done OK)

So that file is processed only on backend, secrets are secure and passed only during continuous deployment stage

Reply:
See screenshot for `api/documents` endpoint specs
![image](https://user-images.githubusercontent.com/31904595/216808305-378ab834-b68e-4046-a9b9-07e0f290760a.png)